### PR TITLE
Hackergarten article

### PR DESCRIPTION
**Pull request by kress04**
Article about what is Hackergarten and why should you join. Written by Tamari and Ixchel during one of the latest Hackergarten Basel events.

### CODE REVIEW

File: _posts/2024-12-12-Hackergarten.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+The goal is to conclude each meeting with a patch or comparable contribution to an openly source and public project. Markus emphasized that contributing to open source does not always involve writing code. A contribution can take various forms, 
+not all of which are technical. This article provides proof of this fact. Marcus Fihlon proposed an outstanding idea, and [Tamari](/people/tamari) and [Ixchel](/people/ixchel) collaborated to execute it.
+
+Within minutes, a list of topics began to take shape. Each person had the opportunity to introduce a project and a first idea of how they would like to contribute. For example, you might have an issue with your favourite open source project you work 
+with or would like to contribute to. We are aware that several open-source projects have spent time welcoming new contributors, and you can find issues marked as "first contribution". However, it is precisely this that makes Hackergarten such a unique event.
```
Comment (madmas) [inline _posts/2024-12-12-Hackergarten.md:33]:
```suggestion
with or would like to contribute to. We are aware that several open-source projects have spent time welcoming new contributors, and you can find issues marked as ["good first contribution"](https://github.com/search?q=label%3A%22good+first+contribution%22+state%3Aopen&type=issues) or ["hackergarten"](https://github.com/search?q=label%3Ahackergarten+state%3Aopen&type=issues). However, it is precisely this that makes Hackergarten such a unique event. 
```

File: _posts/2024-12-12-Hackergarten.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+The goal is to conclude each meeting with a patch or comparable contribution to an openly source and public project. Markus emphasized that contributing to open source does not always involve writing code. A contribution can take various forms, 
+not all of which are technical. This article provides proof of this fact. Marcus Fihlon proposed an outstanding idea, and [Tamari](/people/tamari) and [Ixchel](/people/ixchel) collaborated to execute it.
+
+Within minutes, a list of topics began to take shape. Each person had the opportunity to introduce a project and a first idea of how they would like to contribute. For example, you might have an issue with your favourite open source project you work 
+with or would like to contribute to. We are aware that several open-source projects have spent time welcoming new contributors, and you can find issues marked as "first contribution". However, it is precisely this that makes Hackergarten such a unique event. 
+The event has a clear starting point, people with passion in IT and a willingness to make the world a better place, one contribution at the time.
+
+Here you can see the list of topics we proposed tonight:
```
Comment (madmas) [inline _posts/2024-12-12-Hackergarten.md:36]:
```suggestion
Here you can see the list of topics we proposed:
```

File: _posts/2024-12-12-Hackergarten.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+* Asciidoctor Plugin / Antora Plugin
+* Mastodon setup on (h) Cloud (Bash)
+* Article about Hackergarten for [JavaAktuell](https://www.ijug.eu/de/java-aktuell/zeitschrift/)
```
Comment (madmas) [inline _posts/2024-12-12-Hackergarten.md:40]:
```suggestion
* Article about Hackergarten
```

as it never appeared there i suggest to leave out the reference

File: _posts/2024-12-12-Hackergarten.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+* Apache Solr : 1st contrib issues
+* ZioNSON with DSON-LD (Scala)
+* [JReleaser](https://jreleaser.org)
+* [RDiscount](https://github.com/davidfstr/rdiscount) (Ruby, C)
+* [Pandoc Plant UML Filter](https://github.com/timofurrer/pandoc-plantuml-filter) (Python)
+
+In Hackergarten, there are not many rules. But the intention of the organizer that people split in groups so nobody works alone. So, after a quick vote the project work started.
```
Comment (madmas) [inline _posts/2024-12-12-Hackergarten.md:47]:
```suggestion
In Hackergarten, there are not many rules. But the intention of the organizer is that people split in groups so nobody works alone (and, even if the topic they work on does not "work out", they can at least have fun working together and maybe learn something from each other, e.g. a new IDE shortcut). So, after a quick vote the project work started. 
```

File: _posts/2024-12-12-Hackergarten.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+### How to organise a Hackergarten?
+
+If you are considering organizing a Hackergarten for the first time, it is highly recommended that you attend one as a participant beforehand. This will give you valuable insight into how the event is structured and executed.
+
+To host a **successful** Hackergarten, you will need a suitable venue with ample power plugs as well as reliable WiFi access. It is also important to provide refreshments, snacks, and/or pizza during the event. This will not only keep participants fueled (perhaps even happy) but also create an opportunity for networking and socializing during breaks.
+
+### Social and Communication
+
+To streamline your organization process, a platform like Meetup is highly recommended. It has proven to be effective for many events, including Hackergartens in places such as Basel and Lucerne. And even in Dortmund, Germany, and many other places around the globe. You can easily manage event registration, communication, and updates.
```
Comment (madmas) [inline _posts/2024-12-12-Hackergarten.md:91]:
```suggestion
To streamline your organization process, a platform like Meetup is highly recommended. It has proven to be effective for many events, including Hackergartens in places such as 
[Basel](https://www.meetup.com/hackergarten-basel/), [Zürich](https://www.meetup.com/hackergarten-zurich/events/303915242/) and 
[Lucerne](https://www.meetup.com/hackergarten-luzern/). And even in [Stuttgart](https://www.meetup.com/hackergarten-stuttgart/) and
[Dortmund](https://www.meetup.com/hackergarten-dortmund/), and many other places around the globe. You can easily manage event registration, communication, and updates. There is the [hackergarten.net](https://www.hackergarten.net) Homepage that is intended as a common umbrella for all Hackergarten.
```

**Review by madmas — APPROVED** [review]
some suggestion to improve

**Comment by kress04** [comment]
@madmas : I committed all your changes. Will do the merge right now.

**Pull request merged**